src/main/java/repository/Review.java

```diff
@@ -0,0 +1,6 @@
+package repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+public interface Review extends JpaRepository<Review, Integer> {
```

Comment:
need to add empty line between 3 and 4 line

Reply:
changed